### PR TITLE
Liquibase mongodb guide

### PR DESCRIPTION
**Pull request by loicmathieu**

### CODE REVIEW

File: docs/src/main/asciidoc/liquibase-mongodb.adoc [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    <!-- MongoDB driver dependencies -->
+    <dependency>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-mongodb-client</artifactId>
+    </dependency>
+</dependencies>
+----
+
+Liquibase MongoDB extension support relies on the Quarkus MongoDB client config.
+It didn't support multiple MongoDB client for now.
```
Comment (geoand) [inline docs/src/main/asciidoc/liquibase-mongodb.adoc:57]: I think we can drop this sentence

Comment (geoand) [inline docs/src/main/asciidoc/liquibase-mongodb.adoc:57]:
```suggestion
For the time being, it does not support multiple clients.
```

File: docs/src/main/asciidoc/liquibase-mongodb.adoc [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+# quarkus.liquibase-mongodb.validate-on-migrate=true
+# quarkus.liquibase-mongodb.clean-at-start=false
+# quarkus.liquibase-mongodb.contexts=Context1,Context2
+# quarkus.liquibase-mongodb.labels=Label1,Label2
+# quarkus.liquibase-mongodb.default-catalog-name=DefaultCatalog
+# quarkus.liquibase-mongodb.default-schema-name=DefaultSchema
+----
+
+Add a changeLog file to the default folder following the Liquibase naming conventions: `{change-log}`
+The yaml, json and xml changeLog file formats are also supported.
```
Comment (geoand) [inline docs/src/main/asciidoc/liquibase-mongodb.adoc:82]:
```suggestion
YAML, JSON and XML formats are supported for the changeLog.
```

**Comment by loicmathieu** [comment]
@geoand I apply your suggestions.

**Review by geoand — APPROVED** [review]

**Pull request merged**